docs/initial-milestones.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry: A Roadmap to Convergence
+
+This document covers the intial milestones for each SIG developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the goals and timelines for OpenTelemetry.
+
+If you are interested in starting a new SIG, please do the following:
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new SIG.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+## Initial Milestones
+For languages which have both an OpenTracing and OpenCensus implementation, we would like to achieve parity in OpenTelemetry by **September, 2019**, and sunset the existing OpenTracing and OpenCensus SIGs by **November, 2019**. 
+
+Parity can be defined as the following milestones:
+* A set of interfaces which define the OpenTelemetry Specification in a given programming language.
+* An SDK which can propagate context and record data in common formats.
+* Backwards compatibility with OpenTracing and OpenCensus.
+* Metadata helpers for recording common operations defined in the OpenTelemetry data model.
+* Tests which provide evidence of interoperability.
+* Benchmarks which provide evidence of expected resource utilization.
+
+### SPEC: Define OpenTelemetry Interfaces
+* Metrics interface
+* Tracing interface
+* Tags interface
+
+### SDK: Propagate Common Trace Contexts
+* Propagate context in Trace-Context format
+* Propagate context in custom formats
+
+### SDK: Export Common Data Formats
+* Export data in OpenTelemetry format
+* Export data in custom format
+
+### SDK: Record Common Operations
+* A data structure representing each operation type
+* Utilities to record 
+
+### SDK: OpenTracing Bridge
+* the OpenTelemtry SDK implements the latest version of the OpenTracing interface.
+* register as GlobalTracer.
+
+### SDK: OpenCensus Bridge
+* implement a version of OpenCensus which bridges to OpenTelemetry.
+
+### TEST: Verify Interoperability
+* Integration tests which verify SIG is interoperable with the rest of OpenTelemetry.
```

Comment:
This is ambitious goal. I did some cross-lang test cases for HTTP adapters (auto-collectors): https://github.com/open-telemetry/opentelemetry-specification/blob/master/work_in_progress/opencensus/http-out-test-cases.json We can make it a requirement for all spec-defined mappings, but I doubt everybody have capacity to commit on this. 

Again, maybe this should be beyond the September requirement.

Reply:
Hmmm, maybe there is a simple-but-limited set of integration tests that we can do? I don't want to ship a bug related to trace propagation, data exporting, or metadata definitions. Perhaps I am overly concerned, but I am also old and grizzled; "we'll test it later" has a poor track record in my experience. :)

Reply:
Tend to agree with Ted. Jaeger client libs had cross-language interop tests from day one. It may be even possible to reuse them using OT bridge and exporter to Jaeger data format (although context propagation tests are separate from exporting).

Reply:
We could certainly look at re-using the approach Jaeger took, if you are happy with it and felt it wasn't took much work.

Reply:
I am all for this! My only question - will it fit September timeline. Let's keep this goal and see how we will execute